### ISSUE TITLE
Markdown Processor source?

### ISSUE THREAD

**Issue by benjitusk** [qeude/SwiftDown#14]
Can you release a swift package of the markdown engine? as in, an independent library I can import to use your markdown styling.
For example, I'd like to use something like this:
```swift
import SwiftDownMD

struct Comment: View {
    let comment: Comment
    var body: some View {
        VStack {
            Markdown(comment.body)
        }
    }
}
```

**Comment by qeude** [comment]
Hi @benjitusk,

What would be the use case? It would be to render Markdown as a read only Text?

**Comment by benjitusk** [comment]
Yes, exactly that

**Comment by qeude** [comment]
Isn't this covered by the SwiftUI `Text` component?

**Comment by benjitusk** [comment]
Not quite. your markdown editor has much more extensive support for markdown than SwiftUI `Text`

**Comment by qeude** [comment]
I see, wouldn't a non-editable SwiftDownEditor make the trick?
I think the best solution here would be to recreate the behaviour of `SwiftDownEditor` in an `UILabel` but it might take some time to implement
